src/controller/controller_connectors.cpp

```diff
@@ -271,7 +267,14 @@ namespace Controller{
         JSON::Value p = JSON::fromString(*runningConns.begin());
         buildPipedArguments(p, (char **)&argarr, capabilities);
         // start piped w/ generated args
-        currentConnectors[*runningConns.begin()] = Util::Procs::StartPiped(argarr, 0, 0, &err);
+        // for livepeer-x binaries, pass stdout/stderr straight through the logger
+        if (strncmp(argarr[0], livepeerPrefix.c_str(), strlen(livepeerPrefix.c_str())) == 0) {
+          int my_stdout = 1;
+          int my_stderr = 2;
```

Comment:
Be aware that stderr of the controller is actually connected to the logger. So anything connected there will in fact still go through the log parser.

Reply:
Noted. Maybe I'll just have them both go to stdout.